docs/source/api/index.rst

```diff
@@ -2,344 +2,138 @@
+    account. Attempting to create an object at the other names results in
+    a ``400 Bad Request`` error response.
+
+- Object ETags (MD5 digests) are written as sysmeta, similar to what was
+  done for Swift's encryption and SLO features. If the client provides an
+  ETag header in a PUT request, it will be verified before finalizing the
+  write.
+- Object metadata must be valid UTF-8. Ordinarily, Swift only requires
+  UTF-8 for account and container metadata.
+- Object creation does not support the ``If-None-Match:*`` header.
+- Objects do not support expiration (though the ``X-Delete-At`` and
+  ``X-Delete-After`` headers will be written down).
+
+
+Reading objects via the object API
+----------------------------------
+
+- Account-level container counts, object counts, and total bytes used
+  reflect the storage of the underlying ProxyFS log segments and
+  checkpoints. As such, the counts are not particularly useful and the
+  bytes-used is only a crude approximation.
+- When listing an account, ``delimiter``, ``prefix``, and ``reverse``
+  query params are ignored.
+- When querying containers, the ``X-Storage-Policy`` header returns the
+  default storage policy for the cluster, which is not necessarily the
+  storage policy being used for log segment containers or the checkpoint
+  container.
+- Container-level object counts and total bytes used are always 0.
+- When listing a container, the ``reverse`` query parameter is ignored.
+  Only ``/`` is supported for ``delimiter``.
+- Auto-created directory objects are presented as zero-byte objects
+  with ``Content-Type: application/directory``.
+
+
+Reading objects via the file API
+--------------------------------
+
+- Account, container, and object metadata are inaccessible via the file API.
+- Large objects are only assembled via the object API. DLOs or SLOs, when
+  accessed through the filesystem API, will be zero-byte files or JSON
+  (respectively), similar to what happens when including a
+  ``?multipart-manifest=get`` query parameter via the object API.
+- Swift "symlink" objects are only followed when accessed through the object
+  API. When accessed through the filesystem API, these will be zero-byte
+  files.
+
+Reading files via the object API
+--------------------------------
+
+- Since containers map to directories, files written at the root of the
+  volume are not accessible via the object API.
+- Since files may not have been written sequentially, the ETag *will
+  not* be the MD5 digest of the contents. It should be considered "an
+  opaque validator for differentiating between multiple representations
+  of the same resource" as `RFC 2616
+  <https://tools.ietf.org/html/rfc2616.html#section-13.3.3>`__ and
+  `RFC 7232 <https://tools.ietf.org/html/rfc7232#section-2.3>`__
+  intended.
+
+.. note::
+  Filesystem access to SLO segment data can invalidate the
+  manifest, as the segment ETag may change.
+
+
+Swift functionality broken as of 1.3
```

Comment:
Should probably update the version number. As of 1.9, all of these are still broken (to my knowledge).